tests/basic/storage_pool_test.go

```diff
@@ -384,6 +384,7 @@ func roundUpValue(toRound uint64) uint64 {
 
 func poolResizeIsInProgress(poolToBeResized *api.StoragePool) bool {
 	poolSizeHasBeenChanged := false
+	waitCount := 20
 	if poolToBeResized.LastOperation != nil {
 		for {
```

Comment:
Just wondering why not use already existing `DoRetryWithTimeout()` function we have instead of the doing it via `waitCount`?

Reply:
yes, it can be used, will take it as enhancemet and will update in next PR as i need time to test it